src/components/SearchResults.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from "react";
+
+const SearchResults = ({ bookings }) => {
+  const [selectedCustomerId, setSelectedCustomerId] = useState(null);
+
+  const handleShowProfile = (customerId) => {
+    setSelectedCustomerId(customerId);
+  };
+
+  return (
+    <div>
+      <table className="table table-striped">
+        <thead>
+          <tr>
+            <th>Title</th>
+            <th>First Name</th>
+            <th>Surname</th>
+            <th>Email</th>
+            <th>Room ID</th>
+            <th>Check-in Date</th>
+            <th>Check-out Date</th>
```

Comment:
For task 6 to add the difference of nights You installed moment library which is great. The docs provide some guidance on calculating the difference here https://momentjs.com/docs/#/displaying/difference/